.circleci/config.yml

```diff
@@ -0,0 +1,178 @@
+  node: circleci/node@5.0
+  slack: circleci/slack@4.12.0
+
+executors:
+  mvn-11:
+    docker:
+      - image: 'cimg/openjdk:11.0'
+
+commands:
+  notify-slack:
+    description: "Notify slack on channel << parameters.channel >>"
+    parameters:
+      channel:
+        type: string
+      mentions_failure:
+        type: string
+        default: ''
+      mentions_success:
+        type: string
+        default: ''
+    steps:
+      - slack/notify:
+          event: fail
+          channel: << parameters.channel >>
+          mentions: << parameters.mentions_failure >>
+          template: basic_fail_1
+      - slack/notify:
+          event: pass
+          channel: << parameters.channel >>
+          mentions: << parameters.mentions_success >>
+          template: basic_success_1
+  push-artifacts:
+    description: | 
+      Places the artifacts in in the given repository, to trigger the target pipeline.
+      If the branch is "dev", it will push straight away, if it's trino-snowflake-connector, it will raise a PR.
+    parameters:
+      target_repository:
+        type: string
+      target_branch:
+        type: enum
+        enum: ["dev", "trino-snowflake-connector"]
+      target_path:
+        type: string
+      environment_type:
+        type: enum
+        enum: ["aws", "gcp"]
+    steps:
+      - gh/install
+      - run:
+          name: Clone << parameters.target_repository >> repository
+          command: |
+            branch_name=<< parameters.target_branch >>
+            echo "export branch_name=$branch_name" >> $BASH_ENV
+            git clone https://${GITHUB_TOKEN}@github.com/harbrdata/<< parameters.target_repository >>.git target_repo_<< parameters.environment_type >>
+      - run:
+          name: Copy generated files into new branch << parameters.target_branch >> (<< parameters.target_repository >>)
+          command: |
+            cd target_repo_<< parameters.environment_type >>/<< parameters.target_path >>
+            # Create the branch if it doesn't exist in origin
+            git switch $branch_name || git switch -c $branch_name
+            rm trino-snowflake-*.zip
+            cp /tmp/generated-artifacts/trino-snowflake-*.zip .
+            git add --all
+      - run:
+          name: Push changes to << parameters.target_branch >> (<< parameters.target_repository >>)
+          command: |
+            cd target_repo_<< parameters.environment_type >>/<< parameters.target_path >>
+            if [ -n "$(git status --porcelain)" ]; then
+              echo "Changes to be committed:"
+              git status
+              git config user.email "harbr-installer@harbrdata.com"
+              git config user.name "CircleCI Job [${CIRCLE_PROJECT_REPONAME}]"
+              git commit -m "Automated commit from ${CIRCLE_PROJECT_REPONAME} ${CIRCLE_SHA1} to update trino-snowflake.zip files"
+              git push --set-upstream origin HEAD
+
+              if [ "$branch_name" == "trino-snowflake-connector" ]; then
```

Comment:
Instead of checking for a particular branch, should we instead use a parameter such as "create_pr"?